artifacts/phar/Makefile

```diff
@@ -0,0 +1,20 @@
+# Build phar
+build-phar: test
+	@echo Copying sources
+	@mkdir -p /tmp/phpmetrics-build
+	@cp * -R /tmp/phpmetrics-build
+	@rm -Rf /tmp/phpmetrics-build/vendor /tmp/phpmetrics-build/composer.lock
+
+	@echo Releasing phar
+	@sed -i "s/<VERSION>/`semver tag`/g" /tmp/phpmetrics-build/artifacts/phar/build.php
+
+	@echo Installing dependencies
+	@cd /tmp/phpmetrics-build && composer install --no-dev --optimize-autoloader --prefer-dist
```

Comment:
I fou refer to [this pr](https://github.com/phpmetrics/PhpMetrics/pull/12/files), you see that a bug will occur if `vendor/autoload.php` file exists in the **current directory**. 

To avoid unexpected bugs, I prefer to chdir to another directory.